src/sentry/search/snuba/executors.py

```diff
@@ -228,32 +260,105 @@ def snuba_search(
             ]  # ensure stable sort within the same score
             referrer = "search"
 
-        snuba_results = snuba.aliased_query(
-            dataset=self.dataset,
+        organization = Organization.objects.get(id=organization_id)
+        if not features.has("organizations:performance-issues", organization):
```

Comment:
It might make sense to just remove this code. Instead, we could just skip doing the transaction conditions if this feature is false.

Doing this will be a bit safer, since we're changing all error queries to run on discover when we enable this flag, and I'd like to see the whole test suite run through that path.